README.md

```diff
@@ -1,11 +1,45 @@
 # rectangle-overlap
 
+[![TypeScript support](https://img.shields.io/npm/types/rectangle-overlap.svg)](https://github.com/lovasoa/rectangle-overlap/blob/master/index.ts)
+[![License](https://img.shields.io/npm/l/rectangle-overlap.svg)](https://github.com/lovasoa/rectangle-overlap/blob/master/LICENSE)
+
 Fastly compute the area of the intersection of two rectangles.
 
 ## Usage
 
 ```js
-var area = require("rectangle-overlap");
+const area = require("rectangle-overlap");
+
+let rect1 = {
+  x: 100,
+  y: 100,
+  w: 100,
+  h: 100,
+};
+let rect2 = {
+  x: 50,
+  y: 50,
+  w: 100,
+  h: 100,
+};
+// rectangles overlap partially
+area(rect1.x, rect1.y, rect1.w, rect1.h, rect2.x, rect2.y, rect2.w, rect2.h); // 2500
```

Comment:
I think you forgot to update the function calls. The function takes only two parameters now.